transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -1018,21 +1031,27 @@ JNIEXPORT jint JNICALL Java_io_netty_channel_epoll_Native_finishConnect0(JNIEnv*
     return -optval;
 }
 
-JNIEXPORT jint JNICALL Java_io_netty_channel_epoll_Native_accept0(JNIEnv* env, jclass clazz, jint fd) {
+JNIEXPORT jint JNICALL Java_io_netty_channel_epoll_Native_accept0(JNIEnv* env, jclass clazz, jint fd, jobject acceptedSocket) {
     jint socketFd;
     int err;
+    struct sockaddr_storage addr;
+    socklen_t address_len;
```

Comment:
Depends what c standard we are targeting.  However this will not be the first instance to declare/initialize. see [line 299](https://github.com/netty/netty/pull/3440/files#diff-b3c0218865f48def90464f56f26ecce9R299) and other lines too.